job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy

```diff
@@ -75,7 +75,7 @@ pipeline{
         stage('trigger kogito-tooling nightly job main') {
             steps {
                 build job: 'kogito-tooling.nightly/main', propagate: false, wait: true, parameters: [
-                        [\$class: 'StringParameterValue', name: 'DEPLOYMENT_REPO_URL', value: 'https://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8443/nexus/service/local/repositories/scratch-release-rhba-master/content-compressed'],
+                        [\$class: 'StringParameterValue', name: 'DEPLOYMENT_REPO_URL', value: 'https://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8443/nexus/service/local/repositories/scratch-release-rhba-main/content-compressed'],
```

Comment:
Has this been renamed/created in Nexus along with the scheduled task for clean up?

We might need to adjust the scheduled task ID in QE jobs.

Reply:
https://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8443/nexus/content/repositories/scratch-release-rhba-main/ ??